### DISCUSSION TITLE
Make gists.github.com redirect to gist.github.com

### DISCUSSION THREAD

**Discussion by jnolis** [community/community#5223] [Other Feature Feedback, Questions, & Ideas] [answered]
This is an almost trivial change but it would make my life so much better. I can't ever remember if it's **gists**.github.com or **gist**.github.com and I always get it wrong. Having **gists** redirect would help people like me who can't remember, which very little downside.

**Comment by PythonCoderUnicorn** [comment]
The ask is a simple href tag being added to source code, so it should be a thing

**Comment by emmaviolet** [comment, marked as answer]
Just a note to say this should be sorted for you now!

**Reply by jnolis** [reply]
INCREDIBLE
